src/ensembl/production/dbcopy/models.py

```diff
@@ -359,6 +365,11 @@ def completion(self):
             self.progress
         )
 
+    @property
+    def equivalent_jobs(self):
+        params = {k: getattr(self, k) for k in self.__class__.objects._EQ_PARAMS.keys()}
+        return self.__class__.objects.equivalent_jobs(**params)
```

Comment:
No need of the __class__ call here, it's resolved with the `objects = RequestJobManager()` Which is missing?